internal/engine/analytics/analytics_reporter.go

```diff
@@ -157,5 +162,11 @@ func (ar *AnalyticsReporter) report(ctx context.Context) {
 
 	stats["tiltfile.error"] = tiltfileIsInError
 
+	for k, v := range st.Features {
+		if ar.featureDefaults[k].Status == feature.Active && v != ar.featureDefaults[k].Enabled {
```

Comment:
wait, i'm so confused, i thought @lizzthabet wanted us to have the tag attached even if the default was enabled?

Reply:
whoops, right!